test/srt/test_moe_ep.py

```diff
@@ -0,0 +1,104 @@
+import unittest
```

Comment:
Can you add it to the workflow file so the CI will run this?
https://github.com/sgl-project/sglang/blob/5f12f0e7af585973b366c262a5c3faea5ce5bf0a/.github/workflows/pr-test.yml#L129